### PR TITLE
Default behaviour and membership changes

### PR DESCRIPTION
**Pull request by kgeorgak**

### CODE REVIEW

**Comment by jujugrrr** [comment]
:+1:

**Comment by kgeorgak** [comment]
This PR replaced https://github.com/rackspace-cookbooks/rackspace_users/pull/3

File: Gemfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 group :lint do
   gem 'foodcritic'
   gem 'foodcritic-rackspace-rules'
-  gem 'rubocop'
+  gem 'rubocop', '~> 0.24'
```
Comment (martinb3) [inline Gemfile:6]: I think we can take this out.

**Comment by martinb3** [comment]
:+1: for me. Let's merge and we'll fix the rubocop pin later.

**Pull request merged**